ViewModel/HyvaGrid/ColumnDefinition.php

```diff
@@ -152,4 +157,9 @@ public function isSortable(): bool
     {
         return !isset($this->sortable) || $this->sortable !== 'false';
     }
+
+    public function getIsInitiallyHidden(): bool
```

Comment:
For consistency it would be better to call this method `isInitiallyHidden` (without the `get` prefix).